kuma/wiki/models.py

```diff
@@ -925,6 +929,10 @@ def save(self, *args, **kwargs):
         # Delete any cached last-modified timestamp.
         self.fill_last_modified_cache()
 
+        # Set the shortened URL (e.g. bit.ly).
+        # TODO: Should we do this on a subset of documents? E.g. not redirects?
+        tasks.set_share_url.delay(self.pk)
```

Comment:
We shouldn't do this on every call to `Document.save()` right? That means every time we save a new `Revision` and set it to the document `current_revision`, we will generate a brand new share url for the document?